generatesitemap.py

```diff
@@ -222,6 +222,9 @@ def lastmod(f) :
                     universal_newlines=True).stdout.strip()
     if len(mod) == 0 :
         mod = datetime.now().astimezone().replace(microsecond=0).isoformat()
+    if date_only != "false":
```

Comment:
First see my other comment lower down in file where the argument is first processed. Once that is handled, date_only will be a boolean, so you can eliminate the `!= "false"`.